docs/modules/ROOT/partials/credential-types/x509.adoc

```diff
@@ -39,3 +39,11 @@ In each of the above sections, the following changes MUST be applied:
 * Any reference to the _claim generator_ MUST be replaced with the _<<_actor,actor>>_ whose _X.509 certificate_ is being used for this assertion.
 * The validator SHALL maintain one or more _<<_trust_list,trust lists>>_ which do not need to be the same as the _<<_trust_list,trust lists>>_ used to validate claim generator signatures as described in link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_c2pa_signers++[Section 14.4.1, “C2PA Signers.”]
 * The validator SHALL consider any v1 time-stamp as defined in link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_choosing_the_payload++[Section 10.3.2.5.2, “Choosing the payload”] to be invalid.
+
+=== Display of named actor logo
+
+A *CAWG identity assertion consumer* MAY wish to display an icon or logo for the _<<_named_actor,named actor>>._
+To locate such a graphic, the *CAWG identity assertion consumer* SHALL look inside the embedded certificate for a logotype as defined in https://datatracker.ietf.org/doc/rfc9399/[RFC 9399].
+If no logotype is found in the certificate, the Manifest Consumer MAY use icons or logos from other sources in an implementation-dependent manner.
```

Comment:
There may need to be in the threat language for the UI to differentiate between icons associated with certificates and icons that were included in an "implementation-dependent manner". If it including it in the certificate is meant to denote more trust, then it shouldn't be visibly equivalent to an ad-hoc inclusion.